jwst/tests_nightly/general/nirspec/test_extract2d_msa_8slits.py

```diff
@@ -32,13 +32,13 @@ def test_extract2d_nrs_msa(_bigdata):
         # This is needed because we can't run the step with the
         # MSA configuration file not in the current dir.
         # When this is fixed, the copy can be removed.
-        for f in input_file:
+        for f in input_files:
```

Comment:
Nice catch!  Is this also one that wasn't showing up in pandokia as an error?

Reply:
This test doesn't exist at all on the pandokia side. Where did it come from? Did someone add it as a new test just recently only within Jenkins?

Reply:
This test should be removed. IIRC I tried adding it unsuccessfully to svn some time ago and probably included it up by mistake here.